osu.Game/Screens/SelectV2/Carousel.cs

```diff
@@ -550,6 +550,9 @@ protected override void Update()
                 updateDisplayedRange(range);
             }
 
+            double selectedYPos = currentSelection?.CarouselItem?.CarouselYPosition ?? 0;
+            double maximumDistanceFromSelection = scroll.Panels.Select(p => Math.Abs(((ICarouselPanel)p).DrawYPosition - selectedYPos)).DefaultIfEmpty().Max();
```

Comment:
We can't run linq in `Update` as a rule. Why can't this just be `scroll.Panels[^1]`?